Client/Frontend/Settings/Clearables.swift

```diff
@@ -48,12 +48,17 @@ class CookiesClearable: Clearable {
         let result = Deferred<Maybe<()>>()
         // need event loop to run to autorelease UIWebViews fully
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
-            // Now we wipe the system cookie store (for our app).
+            // Clearing cookies clears both cookies and cache.
             let localStorageClearables: Set<String> = [WKWebsiteDataTypeCookies,
                                                        WKWebsiteDataTypeSessionStorage,
                                                        WKWebsiteDataTypeLocalStorage,
                                                        WKWebsiteDataTypeWebSQLDatabases,
-                                                       WKWebsiteDataTypeIndexedDBDatabases]
+                                                       WKWebsiteDataTypeIndexedDBDatabases,
```

Comment:
Could we extract these cache related items to a helper property? We use it now in two places, here and in `CacheClearable`
Could do it like `fileprivate var localStorageClearables: Set<String> { return [wkwebsite..] }` and place somewhere in the `Clearables.swift` file

Reply:
@jumde @iccub a better way would be to use: `WKWebsiteDataStore.allWebsiteDataTypes()` this returns all above values